plugins/source/okta/client/spec/gen/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"path"
+	"runtime"
+
+	"github.com/cloudquery/cloudquery/plugins/source/okta/client"
+	cqjsonschema "github.com/cloudquery/codegen/jsonschema"
+)
+
+func main() {
+	fmt.Println("Generating JSON schema for plugin spec")
+	cqjsonschema.GenerateIntoFile(new(client.Spec), path.Join(currDir(), "../..", "schema.json"),
```

Comment:
I prefer moving `spec.go` to `spec` pkg, so that all about spec is in a single pkg (schema, too). nit

Reply:
spec-related code here is too short for that, would mean more changes in the codebase in other places.